lib/yt/audit.rb

```diff
@@ -29,5 +29,15 @@ def self.has_subscribe_annotations?(video_id)
         annotation.link && annotation.link[:type] == :subscribe
       end
     end
+
+    # Audit youtube association of a video
+    # @param [String] video_id the video to audit.
+    # @return [Boolean] if the video description has link to its own channel.
+    def self.has_youtube_association?(video_id)
```

Comment:
Yes, the name was used in the report. How about we name the method `has_link_to_own_channel?` and then on the Audit first page "YouTube Association: 10 out of 10"